### PR TITLE
Cust cl/cl list

### PR DESCRIPTION
**Pull request by PiotrekZak**
Includes all changes required for the user to be able to build models with custom classes

### CODE REVIEW

File: zrp/data/processed/standard_population_weights.json [hunk 1]
```diff
@@ -0,0 +1 @@
...K": 0.126243902871607, "AIAN": 0.00823290472845048, "ASIAN": 0.0564095784530982, "PI": 0.00187220288319317, "OTHER": 0.103069869627797}, {"WHITE": 0.582431051175431, "HISPANIC": 0.20265969066824, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "ASIAN": 0.0628918313065141, "PI": 0.00208734529011336}, {"WHITE": 0.522399958663599, "HISPANIC": 0.181771582772255, "BLACK": 0.126243902871607, "AIAN": 0.00823290472845048, "AAPI": 0.0582817813362914, "OTHER": 0.103069869627797}, {"WHITE": 0.582431051175431, "HISPANIC": 0.20265969066824, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "AAPI": 0.0649791765966274}, {"WHITE": 0.704171541435853, "BLACK": 0.126243902871607, "AIAN": 0.00823290472845048, "ASIAN": 0.0564095784530982, "PI": 0.00187220288319317, "OTHER": 0.103069869627797}, {"WHITE": 0.785090741843671, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "ASIAN": 0.0628918313065141, "PI": 0.00208734529011336}, {"WHITE": 0.704171541435853, "BLACK": 0.126243902871607, "AIAN": 0.00823290472845048, "AAPI": 0.0582817813362914, "OTHER": 0.103069869627797}, {"WHITE": 0.785090741843671, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "AAPI": 0.0649791765966274}]
```
Comment (kasey-zest) [inline zrp/data/processed/standard_population_weights.json:1]: Looks like we are missing the two or more races category most of the time. Also looks like we either need a mapping of long names to short names or additional records to this list.

Please add

-  abbreviations for the first item of the list ['WHITE', 'BLACK', 'AIAN', 'ASIAN', 'PI', 'OTHER', 'MULTI']
- additional lists where two or more races & other are separated

Please define the abbreviation for two or more races is it 'multi', 'tom' '2plus' 2ormore', etc

File: zrp/modeling/pipeline_builder.py [hunk 2]
```diff
@@ -185,7 +187,7 @@ class ZRP_DataSampling(BaseZRP):
         Indicates the source of zrp_modeling data to use. There are three optional sources 'block_group', 'census_tract', and 'zip_code'. By default 'census_tract' is inferred.
     """
 
-    def __init__(self, zrp_model_source, file_path=None, zrp_model_name='zrp_0', *args, **kwargs):
+    def __init__(self, zrp_model_source, file_path=None, zrp_model_name='zrp_0', population_weights_dict=None, *args, **kwargs):
```
Comment (kasey-zest) [inline zrp/modeling/pipeline_builder.py:190]: `population_weights_dict`  is missing from the doc string, please add a description including details on how someone should add custom population weights

File: zrp/modeling/pipeline_builder.py [hunk 3]
```diff
@@ -280,14 +272,72 @@ def validate_input_columns(self, data):
...
+        user_target_classes = set(data[self.race].unique())
+        for standard_population_weights in standard_population_weights_dicts:
+            if user_target_classes == set(standard_population_weights.keys()):
+                return standard_population_weights
+    
     def fit(self):
         return self
 
-    def transform(self, data):
+    def transform(self, data, population_weights_dict = None):
```
Comment (kasey-zest) [inline zrp/modeling/pipeline_builder.py:329]: missing docstring

**Review by kasey-zest — CHANGES_REQUESTED** [review]
please resolve comments, prior to merging

File: zrp/data/processed/standard_population_weights.json (hunk 1 above)
Comment (PiotrekZak) [reply zrp/data/processed/standard_population_weights.json:1]: New version includes all of the corrections mentioned above. I decided to keep short versions only - the long ones seem to add extra level of complexity for no reason imo.

File: zrp/modeling/pipeline_builder.py (hunk 2 above)
Comment (PiotrekZak) [reply zrp/modeling/pipeline_builder.py:190]: done

File: zrp/modeling/pipeline_builder.py (hunk 3 above)
Comment (PiotrekZak) [reply zrp/modeling/pipeline_builder.py:331]: done

**Review by kasey-zest — APPROVED** [review]

**Pull request merged**